Makefile

```diff
@@ -3,10 +3,10 @@
 # To re-generate a bundle for another specific version without changing the standard setup, you can:
 # - use the VERSION as arg of the bundle target (e.g make bundle VERSION=0.0.2)
 # - use environment variables to overwrite this value (e.g export VERSION=0.0.2)
-VERSION ?= 3.11.1
+VERSION ?= 3.11.1-2
 # Replaces Operator version
 # Set this when when there is a new patch release in the channel.
-REPLACES_VERSION ?= 0.2.6
```

Comment:
I think REPLACES_VERSION stays the same.

Reply:
Okay--I wasn't sure on how the versioning worked--I'll revert it.